frontend/src/pages/CompareV2.tsx

```diff
@@ -339,9 +341,41 @@ function CompareV2(props: CompareV2Props) {
     staleTime: Infinity,
   });
 
+  // Ensure that the two-panel selected artifacts are present in selected valid run list.
+  const getVerifiedTwoPanelSelection = (
```

Comment:
This method verifies that the existing two-panel selection (Confusion Matrix, HTML, and Markdown) is from a valid and currently-selected run; it does so in an extendable manner that would fit with adding more columns to this panel layout.